client/src/pages/UpgradePage.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import NavBar from "../components/Header/NavBar";
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import UpgradeCard from "../components/upgradeCard/UpgradeCard";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: "inherit",
+    flexGrow: 1
+  },
+  main: {
+    display: "flex",
+    flexDirection:"column",
+    marginTop: theme.spacing(6)
+  },
+  heading: {
+    textAlign: "center"
+  },
+  title: {
+    fontWeight: 600,
+    marginBottom: theme.spacing(1)
+  },
+  subtitle: {
+    fontSize: 16,
+    color: "#909090"
+  },
+  cardContainer:{
+    marginTop:theme.spacing(4)
+  }
+}));
+
+const UpgradePage = () => {
+  const classes = useStyles();
+  const plan= "free basic";
```

Comment:
I'm assuming the plan is to get this value from the back-end when it's integrated in a future PR?